src/predictor/gpu_predictor.cu

```diff
@@ -373,7 +373,7 @@ class GPUPredictor : public xgboost::Predictor {
   }
 
  public:
-  GPUPredictor()
+  GPUPredictor()  // NOLINT
```

Comment:
I'm curious what's the error message.

Reply:
Something about using `=default` for default constructors, I think it was a false positive.